code/datums/gamemode/powers/changeling.dm

```diff
@@ -226,6 +226,7 @@
 		return
 	var/mob/living/carbon/human/C = R.antag.current
 	to_chat(C, "<span class='notice'>We distort our form to prevent AI-tracking.</span>")
+	C.digitalcamo = 1
```

Comment:
hang on the digital camo didn't even work? hahahahaha

Reply:
He removed digital camo and put it under a single giant proc that was meant to handle every single kind of anti-tracking, which went about as well as you'd expect. I re-added the digital camo variable and undid his changes to the proc.

Reply:
![image](https://github.com/vgstation-coders/vgstation13/assets/16082007/281d139c-d883-4efb-8d50-bcbc586d62eb)